.github/workflows/preview-theme.yml

```diff
@@ -1,8 +1,8 @@
 name: Theme preview
 
 on:
-  pull_request_target:
-    types: [opened, synchronize, reopened]
+  pull_request:
+    types: [opened, edited, reopened, synchronize]
```

Comment:
For some reason, the `synchronize` event is not triggered when a pull request is closed, and the user adds a commit. I have to look at how this can be done.